superset-frontend/packages/superset-ui-core/test/chart/components/SuperChart.test.tsx

```diff
@@ -18,12 +18,14 @@
  */
 
 import React from 'react';
-import { mount } from 'enzyme';
 import mockConsole, { RestoreConsole } from 'jest-mock-console';
 import { triggerResizeObserver } from 'resize-observer-polyfill';
 import ErrorBoundary from 'react-error-boundary';
 
 import { promiseTimeout, SuperChart } from '@superset-ui/core';
+// @ts-ignore
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I'm not sure if we should create this dependency. Especially because we're moving the files in `spec` to `src`. Maybe let's just copy `styledMount` logic for now while we think about cyclic dependencies.